sources/packages/backend/libs/integrations/src/atbc-integration/atbc-integration.processing.service.ts

```diff
@@ -45,7 +45,7 @@ export class ATBCIntegrationProcessingService {
     if (response) {
       const auditUser = await this.systemUsersService.systemUser();
       // Update PD sent date.
-      await this.studentService.updatePDSentDate(student.id, auditUser);
+      await this.studentService.updatePDRequested(student.id, auditUser);
```

Comment:
since you already updated the method name, maybe you can use `disability status` here too instead of `pd`